core/consensus/babe/primitives/src/lib.rs

```diff
@@ -18,6 +18,7 @@
 #![deny(warnings)]
 #![forbid(unsafe_code, missing_docs, unused_variables, unused_imports)]
 #![cfg_attr(not(feature = "std"), no_std)]
+#![allow(clippy::too_many_arguments)]
```

Comment:
can you to scope it down to individual methods/functions instead?

Reply:
This lint is triggered by some function declared in `#![deny(warnings)]` and `decl_runtime_apis!` but it is now being removed to avoid `clippy` attributes